AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -1879,6 +1879,8 @@ crisisgroup.org##.c-cta-block
 thenation.com##.indent.current-issue
 thenation.com##.inline-cta-1
 examine.com##.optin-wrapper
+currentaffairs.org##.wp-block-image:has(> a[href$="/subscribe"])
+currentaffairs.org##.wp-block-image:has(> a[href$="/donate"])
```

Comment:
I don't see that. Where do you need it?

Reply:
e.g. `https://www.currentaffairs.org/2022/08/our-invasions`

screenshot:
<details>

<img width="1432" alt="image" src="https://user-images.githubusercontent.com/110956608/187924070-d789348e-8bd5-4f60-8fe9-faa79be5a2db.png">

</details>

Reply:
Ok, thanks. You can also add full link here.
1. This rule should be written with `#?#` syntax `currentaffairs.org##.wp-block-image:has(> a[href="http://currentaffairs.org/subscribe"])`.
2. Move it to widgets - donation section.

Reply:
unfortunately can't be full URL because sometimes it is:
- `http://currentaffairs.org/subscribe`
- `https://www.currentaffairs.org/subscribe`
- `https://currentaffairs.org/subscribe`

Reply:
Hm, then leave it as it is, but the syntax should be `#?#`.
`##` only for basic rules.

Reply:
ok, will update the PR based on your feedback